Makefile

```diff
@@ -49,7 +49,7 @@ CC = $(CROSS_COMPILE)gcc
 LD = $(CC)
 WARNINGS = -Wall
 BASE_FLAGS = -fPIC -fvisibility=hidden
-DEFINES = -DNFC_PLUGIN_EXTERNAL
+override DEFINES := -DNFC_PLUGIN_EXTERNAL $(DEFINES)
```

Comment:
DEFINES += -DNFC_PLUGIN_EXTERNAL

Reply:
Thanks for the review. That part was reworked.